lib/stack_master/commands/apply.rb

```diff
@@ -64,7 +67,8 @@ def create_stack
           failed!("Stack creation aborted")
         end
         upload_files
-        cf.create_stack(stack_options.merge(tags: proposed_stack.aws_tags))
+        on_failure = @config.stack_defaults['on_failure'] || @options.on_failure
```

Comment:
You're right when you call the CLI command it passes a `Commander::Command::Options` object, but inside our specs we use a vanilla Hash.